service/resource/s3workercloudconfigv1/delete.go

```diff
@@ -0,0 +1,28 @@
+package s3workercloudconfigv1
+
+import (
+	"context"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/operatorkit/framework"
+)
+
+func (r *Resource) ApplyDeleteChange(ctx context.Context, obj, deleteChange interface{}) error {
+	return nil
+}
+
+func (r *Resource) NewDeletePatch(ctx context.Context, obj, currentState, desiredState interface{}) (*framework.Patch, error) {
```

Comment:
See my other comment. I don't think we should implement deleting of objects.

Reply:
Indeed, They were there just for implementing the interface, but your suggestion makes total sense, I've moved the stub methods to `resource.go`.